i2c/devices/ds3231/README.md

```diff
@@ -0,0 +1,4 @@
+## DS3231
+
+This example makes use of the DS3231 RTC.
```

Comment:
Do you mind also including the manufacturers name to make it easier for people to search it up? Just like how for the PN532 we refer to it as the "NXP PN532 card reader" since it's made by NXP.

Reply:
done